modules/Microsoft.VirtualMachineImages/imageTemplates/readme.md

```diff
@@ -370,7 +370,7 @@ module imageTemplates './Microsoft.VirtualMachineImages/imageTemplates/deploy.bi
       "value": "CanNotDelete"
     },
     "managedImageName": {
-      "value": "<<namePrefix>>-az-mi-x-001"
+      "value": "<<namePrefix>>-az-mi-x-vmicom-001"
```

Comment:
After changing the resource names as suggested above, the readme file will need to be updated through the utility

Reply:
ok